m/mtr/stone.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+# SPDX-FileCopyrightText: © 2025-     AerynOS OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : mtr
+version     : "0.96"
+release     : 1
+homepage    : https://www.bitwizard.nl/mtr/
+upstreams   :
+    - https://github.com/traviscross/mtr/archive/refs/tags/v0.96.tar.gz : 73e6aef3fb6c8b482acb5b5e2b8fa7794045c4f2420276f035ce76c5beae632d
```

Comment:
https://www.bitwizard.nl/mtr/files/

Reply:
> ## Where to get MTR
>
> mtr is now maintained in[ a git repository at github](https://github.com/traviscross/mtr).
> 
> Alternatively you can get MTR from the BitWizard web site at[ https://www.bitwizard.nl/mtr/files](https://www.bitwizard.nl/mtr/files). (The FTP site has been turned off a while ago.) I'm still struggling to make my new "make a release script" adapt to the new situation with git. If the latest version is missing, and you can't get it from github for one reason or another, let me know and I'll have to make one for you manually.

Hence, it seems like GH is the more reliable source, which is why I selected that.

Reply:
Yes but they've already run the autoconf stuff on the tarballs available from that source hosting.